.github/workflows/jenkins-security-scan.yml

```diff
@@ -0,0 +1,21 @@
+name: Jenkins Security Scan
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    types: [ opened, synchronize, reopened ]
+  workflow_dispatch:
+
+permissions:
+  security-events: write
+  contents: read
+  actions: read
+
+jobs:
+  security-scan:
+    uses: jenkins-infra/jenkins-security-scan/.github/workflows/jenkins-security-scan.yaml@v2
+    with:
+      java-cache: 'maven' # Optionally enable use of a build dependency cache. Specify 'maven' or 'gradle' as appropriate.
+      # java-version: 21 # Optionally specify what version of Java to set up for the build, or remove to use a recent default.
```

Comment:
I haven't verified this plugin with Java 17. Perhaps better to say `java-version: 11` for now?

Reply:
Since you haven’t overridden the `java.version` in your `pom.xml` it should not matter as the build will produce Java 11 bytecode anyways. Relying on the default setting makes life - and maintaining - a little easier.

Reply:
Since you haven’t overridden the `java.version` in your `pom.xml` it should not matter as the build will produce Java 11 bytecode anyways. Relying on the default setting makes life - and maintaining - a little easier.

Reply:
Okay, thanks!